docs/install.rst

```diff
@@ -239,7 +239,7 @@ Windows binaries
 
 You can download the latest stable Windows installer from the `GitHub Releases Page <https://github.com/streamlink/streamlink/releases/latest>`__.
 
-Alternatively, you can download the latest `nightly Windows installer <https://streamlink-builds.s3.amazonaws.com/nightly/windows/streamlink-latest.exe>`__.
+Alternatively, you can download the latest `nightly Windows installer <https://dl.bintray.com/streamlink/streamlink-nightly/streamlink-latest.exe>`__.
```

Comment:
You said that you had a monthly traffic of 133GiB. Maybe that's because we don't properly mention here that nightly builds are potentially unstable and only meant for testing purposes. I think this could be improved to something like

> A Windows installer of the latest **stable release** can be found on the Github releases page.
> Alternatively, a Windows installer of latest **development build** for testing purposes can be found here.

I'm not sure, but I think that directly linking to the "-latest" installer is bad. People usually copy those links and paste them on other websites and people clicking on those links don't know what exactly they are downloading. Linking to the file list should make people more aware of the fact that those are dev builds.